routemaster/lua/nack.lua

```diff
@@ -0,0 +1,12 @@
+--
+-- Negative-acknowledges a message.
+--
+-- KEYS[1]: list of new message UUIDs
+-- KEYS[2]: zset of pending (nacked) message UUIDs
```

Comment:
I think it's OK tbh, it's only internal code really so as long as the right keys/args are being sent then it shouldn't be an issue.